README.md

```diff
@@ -25,7 +25,7 @@ The `ark` resource will:
 
 By default, the ark will not run again if the `:path` is not empty. Ark provides many actions to accommodate different use cases, such as `:dump`, `:cherry_pick`, `:put`, and `:install_with_make`.
 
-At this time ark only handles files available from URLs using the [remote_file](http://docs.chef.io/resource_remote_file.html) provider. It does handle local files using the `file://` protocol.
```

Comment:
You can just take the "at this time" bit out and provider should be resource. Provider is the Chef 11 naming of things.